web/yo/app/scripts/controllers/reportgenerator.js

```diff
@@ -204,4 +204,264 @@ angular.module('oncokbApp')
                     });
                 }
             };
+            $scope.reviewedDT = {};
+            $scope.reviewedDT.dtOptions = {
+                paging: true,
+                hasBootstrap: true,
+                scrollY: 500,
+                scrollCollapse: true
+            };
+            $scope.evidenceType = '';
+            $scope.evidenceTypes = [{
+                label: 'Gene Type',
+                value: 'gene_type'
+            }, {
+                label: 'Mutation Effect',
+                value: 'mutation_effect'
+            }, {
+                label: 'Tumor Summary',
+                value: 'tumor_summary'
+            }, {
+                label: 'Drugs',
+                value: 'drugs'
+            }];
+            $scope.reviewedDataHeaders = {
+                gene_type: ['Gene', 'Oncogene', 'Tumor Suppressor'],
+                mutation_effect: ['Gene', 'Mutation', 'Oncogenic', 'Mutation Effect', 'Description', 'Citations'],
+                tumor_summary: ['Gene', 'Mutation', 'Tumor Type', 'Tumor Summary'],
+                drugs: ['Gene', 'Mutation', 'Tumor Type', 'Drugs', 'Level', 'Description', 'Citations']
+            };
+            var reviewedData = {
```

Comment:
Why we call it `reviewedData`?

Reply:
Because it is designed to hold reviewed data